HealthcareSystem/Backend/Service/SendingMail/MailService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Backend.Model.Users;
+using Backend.Settings;
+using MailKit.Net.Smtp;
+using MailKit.Security;
+using Microsoft.Extensions.Options;
+using MimeKit;
+
+namespace Backend.Service.SendingMail
+{
+    public class MailService : IMailService
+    {
+        private readonly MailSettings _mailSettings;
+
+        public MailService(IOptions<MailSettings> mailSettings)
+        {
+            _mailSettings = mailSettings.Value;
+        }
+        public async Task SendWelcomeEmailAsync(WelcomeRequest request)
+        {
+            string FilePath = Directory.GetCurrentDirectory() + "\\Templates\\WelcomeTemplate.html";
```

Comment:
I really like that you have a template for this for flexibility, but should we maybe pull the path to the templates into app.config? @vlaksi what do you think?